benchmarks/policy_performance/policy_perf_test.hpp

```diff
@@ -376,14 +376,15 @@ void test_policy(int team_range, int thread_range, int vector_range,
     // parallel_reduce RangePolicy: range = team_size*team_range
     if (test_type == 400) {
       Kokkos::parallel_reduce(
-          "400 outer reduce", team_size * team_range,
+          "400 outer reduce", static_cast<size_t>(team_size) * team_range,
           KOKKOS_LAMBDA(const int idx, double& val) { val += idx; }, result);
       result_expect =
           0.5 * (team_size * team_range) * (team_size * team_range - 1);
```

Comment:
```suggestion
          0.5 * (team_size * team_range) * (team_size * team_range - 1);
```
I'm surprised it doesn't complain here.

Reply:
It only warns to widening to an integral type as I far as I know